modules/EnsEMBL/Web/Component/Transcript.pm

```diff
@@ -21,7 +21,7 @@ package EnsEMBL::Web::Component::Transcript;
 
 use strict;
 
-use parent qw(EnsEMBL::Web::Component);
```

Comment:
I dont think thats the right fix for this. `Component.pm` is still the parent for this Module not shared.pm

`Shared.pm` is another child of `Component.pm` like `Regulation.pm` or `Variation.pm`. One fix that i can think of is move `get_matches` to `Component.pm` but this will need to look into more as to why this function is missing.